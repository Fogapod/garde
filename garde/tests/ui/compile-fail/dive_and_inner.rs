#![allow(dead_code)]

#[derive(garde::Validate)]
struct Test<'a> {
    #[garde(dive, inner(length(min = 1)))]
    field: &'a [&'a str],
}

fn main() {}
