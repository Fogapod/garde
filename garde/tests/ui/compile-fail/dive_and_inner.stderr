error: `dive` may not be combined with `inner`
 --> tests/ui/compile-fail/dive_and_inner.rs
  |
  |     #[garde(dive, inner(length(min = 1)))]
  |             ^^^^
